lib/cylc/rundb.py

```diff
@@ -1023,15 +1016,11 @@ def upgrade_pickle_to_json(self):
                     else:
                         args.append(cell)
             except (EOFError, TypeError, LookupError, ValueError):
-                n_skips += 1  # skip bad rows
+                pass
```

Comment:
These were diagnostics for my own use when I wrote these upgrade methods. 

They are mostly upgrade methods for restarting Cylc 6 suites in Cylc 7. We should not have much usage for them any more. Happy to revert and leave them alone but aim to remove them for Cylc 8.

Reply:
I think it's OK to remove them, as they were for testing/debugging only.

When we go full Python3, this - and other try/catch with similar pattern - can go away too with

```python
with ignored(EOFError, TypeError, LookupError, ValueError):
    for col, cell in zip(cols, row):
        ...
```

Reply:
Cool syntax.